File: src/server/lib/java/org/meet4xmas/wire/Response.java
package org.meet4xmas.wire;

import org.meet4xmas.util.StringUtil;

public class Response implements java.io.Serializable {
  public boolean success;
  public ErrorInfo error;
  public Object payload;

  public String toString() {
    StringBuilder sb = new StringBuilder("ResponseBody(");
    sb.append("@success: ").append(success).append(", ");
    sb.append("@error: ").append(StringUtil.ValueOrNullToString(error)).append(", ");
    sb.append("@payload: ").append(StringUtil.ValueOrNullToString(payload));
    sb.append(")");
    return sb.toString();
  }
}

File: src/wp7/Meet4Xmas/org/meet4xmas/wire/Response.cs
using System.Text;

namespace org.meet4xmas.wire
{
    public class Response
    {
        public bool success;
        public ErrorInfo error;
        public object payload;

        public override string ToString()
        {
            StringBuilder sb = new StringBuilder("<Response: ");
            sb.Append("@success: ").Append(success).Append(", ");
            sb.Append("@error: ").Append(error.ToString()).Append(",");
            sb.Append("@payload: ").Append(payload.ToString()).Append(">");
            return sb.ToString();
        }
    }
}
